core/main.go

```diff
@@ -33,13 +33,13 @@ func initDependencies() (*dependencies, error) {
 	}
 
 	// init services db.
-	serviceDB, err := database.NewServiceDB(filepath.Join(config.Core.Path, config.Core.Database.ServiceRelativePath))
+	serviceDB, err := database.NewServiceDB(filepath.Join(config.Path, config.Database.ServiceRelativePath))
```

Comment:
config.Database.ServiceRelativePath should already contain config.Path

Reply:
At the end it makes sense to keep `config.Path` out of `config.Database.XXXXRelativePath` so by only modifiying `config.Path`, it will change all other path that use this folder.
The user doesn't need to know all sub-folder.

Reply:
What I want to avoid is the fact that other package needs to modify configs path. In that case, only config package manages paths etc... What you wrote is true also when config package joins paths.

To be clear I don't want a user to specify the full path for the database. I want move join into config package.